sdk/cognitiveservices/cognitiveservices-luis-runtime/README.md

```diff
@@ -17,54 +17,52 @@ npm install @azure/cognitiveservices-luis-runtime
 
 #### nodejs - Authentication, client creation and getVersionPrediction prediction as an example written in TypeScript.
 
-##### Install @azure/ms-rest-nodeauth
+##### Install @azure/ms-rest-azure-js
 
 ```bash
-npm install @azure/ms-rest-nodeauth
+npm install @azure/ms-rest-azure-js
 ```
 
 ##### Sample code
 
 ```typescript
-import * as msRest from "@azure/ms-rest-js";
-import * as msRestNodeAuth from "@azure/ms-rest-nodeauth";
-import { LUISRuntimeClient, LUISRuntimeModels, LUISRuntimeMappers } from "@azure/cognitiveservices-luis-runtime";
-const subscriptionId = process.env["AZURE_SUBSCRIPTION_ID"];
-
-msRestNodeAuth.interactiveLogin().then((creds) => {
-  const client = new LUISRuntimeClient(creds, subscriptionId);
-  const appId = ec7b1657-199d-4d8a-bbb2-89a11a42e02a;
-  const versionId = "testversionId";
-  const predictionRequest: LUISRuntimeModels.PredictionRequest = {
+import { CognitiveServicesCredentials } from "@azure/ms-rest-azure-js";
+import { LUISRuntimeClient } from "@azure/cognitiveservices-luis-runtime";
+
+let subscriptionId = "<luis-authoring-key>";
```

Comment:
Is `subscriptionId` the right name to use here?
I know that all Cognitive Services will be using the `CognitiveServicesCredentials` which in turn refers to this as `subscriptionKey`.

Which of the 2 makes more sense?

Reply:
@chrishmsft What are your thoughts here?

Reply:
I'd suggest 'Subscription Key' to be consistent with all of the other Cognitive Services. We do call this key the 'authoring key' in luis.ai portal so both will be a little inconsistent.